service/controller/resource/deployment/publiips.go

```diff
@@ -0,0 +1,27 @@
+package deployment
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+)
+
+func (r Resource) getCPPublicIPAddresses(ctx context.Context) ([]string, error) {
```

Comment:
When are these public IP's created? Could they be passed as parameters to the azure-operator instead of fetching them? or do they change? That way I could pass whatever I want when running locally.

Reply:
they're created (today) by terraform at control plane deployment.
I don't think they should be passed. That would mean hardcode them in the `installations` repo.